packages/apollo/src/commands/service/check.ts

```diff
@@ -259,6 +263,10 @@ export default class ServiceCheck extends ProjectCommand {
     try {
       await this.runTasks<TasksOutput>(
         ({ config, flags, project }) => {
+          if (!isServiceProject(project)) {
```

Comment:
I love this, but can we make the error message a bit easier to understand / more actionable from the user's perspective? They may not know what they've gotten themselves into. We should let them know we've found a client project, but need a service project. We've got a bitly link scattered throughout the repo for pointing users to the apollo config docs that would be helpful here: https://bit.ly/2ByILPj

Reply:
Great point. I imagined this only being for development (to catch myself); I guess a user could get here too and needs a way to get out.